koku/masu/external/downloader/aws/aws_report_downloader.py

```diff
@@ -237,7 +241,10 @@ def download_file(self, key, stored_etag=None, manifest_id=None, start_date=None
                 msg = "Unable to find {} in S3 Bucket: {}".format(s3_filename, self.report.get("S3Bucket"))
                 LOG.info(log_json(self.tracing_id, msg, self.context))
                 raise AWSReportDownloaderNoFileError(msg)
-
+            if ex.response["Error"]["Code"] == "AccessDenied":
+                msg = "Unable to access S3 Bucket {}: (AccessDenied)".format(self.report.get("S3Bucket"))
+                LOG.info(log_json(self.tracing_id, msg, self.context))
+                raise AWSReportDownloaderAccessDenied(msg)
```

Comment:
question: Do we need to catch this new `AWSReportDownloaderAccessDenied` exception anywhere?

Reply:
I went over it again with a fine tooth comb, and found one other place we could use it.

Reply:
What I mean is, where do we catch the `AWSReportDownloaderAccessDenied` exception? I assume it would need to go [here](https://github.com/project-koku/koku/blob/3feca123c54d5de8aeab5944ba2e4306245b7ad4/koku/masu/external/downloader/aws/aws_report_downloader.py#L171) or [here](https://github.com/project-koku/koku/blob/3feca123c54d5de8aeab5944ba2e4306245b7ad4/koku/masu/external/report_downloader.py#L250) or somewhere, right?